### PR TITLE
Plat 249 remove limit on bulk rerun

### PR DESCRIPTION
**Pull request by bradsawadye**

### CODE REVIEW

File: app/scripts/controllers/transactions.js [hunk 1]
```diff
@@ -413,18 +413,23 @@ export function TransactionsCtrl ($scope, $uibModal, $location, $timeout, $inter
 
   const refreshSuccess = function (transactions) {
     // on success
-    $scope.transactions = transactions
+    if (transactions.count) {
```
Comment (marrouchi) [inline app/scripts/controllers/transactions.js:416]:
```suggestion
    if (transactions.length) {
```
Is transactions a object or an array ?

File: app/scripts/controllers/transactions.js [hunk 2]
```diff
@@ -413,18 +413,23 @@ export function TransactionsCtrl ($scope, $uibModal, $location, $timeout, $inter
...
+      $scope.transactions = []
+      $scope.transactionsCount = transactions.count
+    } else {
+      $scope.transactions = transactions
```
Comment (marrouchi) [inline app/scripts/controllers/transactions.js:420]: `transactionsCount` should be updated here as well right ?

**Review by marrouchi — CHANGES_REQUESTED** [review]
I don't have background on the intent of the PR, so I did a review with the best of my knowledge :)

File: app/scripts/controllers/transactions.js (hunk 1 above)
Comment (bradsawadye) [reply app/scripts/controllers/transactions.js:416]: The endpoint can return an array of transactions or an object containing the count property, which is the length of the transactions to be rerun (for bulk reruns).

Comment (marrouchi) [reply app/scripts/controllers/transactions.js:416]: Thanks for the clarification

**Review by marrouchi — APPROVED** [review]

**Review by michaelloosen — APPROVED** [review]
LGTM

**Pull request merged**